tests/tck/features/match/SeekByEdge.feature

```diff
@@ -0,0 +1,409 @@
+      | "Jason Kidd"         | "Steve Nash"         | "Lakers"        |
+      | "Jason Kidd"         | "Steve Nash"         | "Mavericks"     |
+      | "Jason Kidd"         | "Steve Nash"         | "Suns"          |
+      | "Jason Kidd"         | "Dirk Nowitzki"      | "Mavericks"     |
+      | "Jason Kidd"         | "Vince Carter"       | "Hawks"         |
+      | "Jason Kidd"         | "Vince Carter"       | "Kings"         |
+      | "Jason Kidd"         | "Vince Carter"       | "Magic"         |
+      | "Jason Kidd"         | "Vince Carter"       | "Raptors"       |
+      | "Jason Kidd"         | "Vince Carter"       | "Suns"          |
+      | "Jason Kidd"         | "Vince Carter"       | "Nets"          |
+      | "Jason Kidd"         | "Vince Carter"       | "Mavericks"     |
+      | "Jason Kidd"         | "Vince Carter"       | "Grizzlies"     |
+      | "Dirk Nowitzki"      | "Jason Kidd"         | "Nets"          |
+      | "Dirk Nowitzki"      | "Jason Kidd"         | "Suns"          |
+      | "Dirk Nowitzki"      | "Jason Kidd"         | "Mavericks"     |
+      | "Dirk Nowitzki"      | "Jason Kidd"         | "Knicks"        |
+      | "Dirk Nowitzki"      | "Jason Kidd"         | "Mavericks"     |
+      | "Dirk Nowitzki"      | "Dwyane Wade"        | "Heat"          |
+      | "Dirk Nowitzki"      | "Dwyane Wade"        | "Cavaliers"     |
+      | "Dirk Nowitzki"      | "Dwyane Wade"        | "Bulls"         |
+      | "Dirk Nowitzki"      | "Dwyane Wade"        | "Heat"          |
+      | "Dirk Nowitzki"      | "Steve Nash"         | "Suns"          |
+      | "Dirk Nowitzki"      | "Steve Nash"         | "Lakers"        |
+      | "Dirk Nowitzki"      | "Steve Nash"         | "Mavericks"     |
+      | "Dirk Nowitzki"      | "Steve Nash"         | "Suns"          |
+      | "Paul George"        | "Russell Westbrook"  | "Thunders"      |
+      | "Grant Hill"         | "Tracy McGrady"      | "Spurs"         |
+      | "Grant Hill"         | "Tracy McGrady"      | "Magic"         |
+      | "Grant Hill"         | "Tracy McGrady"      | "Rockets"       |
+      | "Grant Hill"         | "Tracy McGrady"      | "Raptors"       |
+      | "Shaquile O'Neal"    | "Tim Duncan"         | "Spurs"         |
+      | "Shaquile O'Neal"    | "JaVale McGee"       | "Lakers"        |
+      | "Shaquile O'Neal"    | "JaVale McGee"       | "Warriors"      |
+      | "Shaquile O'Neal"    | "JaVale McGee"       | "Wizards"       |
+      | "Shaquile O'Neal"    | "JaVale McGee"       | "Nuggets"       |
+      | "Shaquile O'Neal"    | "JaVale McGee"       | "Mavericks"     |
+
+  Scenario: seek by edge without index
+    When executing query:
+      """
+      MATCH (p1)-[:like]->(p2)
+      RETURN p1.name, id(p2)
+      """
+    Then a ExecutionError should be raised at runtime: Can't solve the start vids from the sentence: MATCH (p1)-[:like]->(p2) RETURN p1.name,id(p2)
```

Comment:
Will we support this statement in the future? Neo4j is supported.

Reply:
like has no index.